src/components/page/StandardPage.tsx

```diff
@@ -396,49 +393,54 @@ const StandardPageInner = <T,>({
-                fieldFilters={fields
-                  .filter(({ filter }) => filter && !filter.primary && !filter.standalone)
-                  .map(toFieldFilter(flatData))}
-                onFilterUpdate={setSelectedFilters}
-                selectedFilters={selectedFilters}
-                supportedFilterTypes={supportedFilters}
-              />
-              {Boolean(fields.find((field) => field.filter?.standalone)) && (
-                <FilterGroup
+
+              <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">
+                {primaryFilters.length > 0 && (
+                  <FilterGroup
+                    fieldFilters={primaryFilters}
+                    onFilterUpdate={setSelectedFilters}
+                    selectedFilters={selectedFilters}
+                    supportedFilterTypes={supportedFilters}
+                  />
+                )}
+                <AttributeValueFilter
                   fieldFilters={fields
-                    .filter((field) => field.filter?.standalone)
+                    .filter(({ filter }) => filter && !filter.primary && !filter.standalone)
                     .map(toFieldFilter(flatData))}
                   onFilterUpdate={setSelectedFilters}
                   selectedFilters={selectedFilters}
                   supportedFilterTypes={supportedFilters}
                 />
-              )}
-              <ManageColumnsToolbar
-                resourceFields={fields}
-                defaultColumns={fieldsMetadata}
-                setColumns={setFields}
-              />
-              {GlobalActionToolbarItems?.length > 0 &&
-                GlobalActionToolbarItems.map((Action, index) => (
-                  <Action key={index} dataOnScreen={showPagination ? pageData : filteredData} />
-                ))}
-            </ToolbarToggleGroup>
-
-            {selectedCountLabel && (
-              <ToolbarItem className="forklift-page__toolbar-item__selected-count">
-                {selectedCountLabel(selectedIds.length ?? 0)}
-              </ToolbarItem>
-            )}
+                {!!fields.find((field) => field.filter?.standalone) && (
+                  <FilterGroup
+                    fieldFilters={fields
+                      .filter((field) => field.filter?.standalone)
+                      .map(toFieldFilter(flatData))}
+                    onFilterUpdate={setSelectedFilters}
+                    selectedFilters={selectedFilters}
+                    supportedFilterTypes={supportedFilters}
+                  />
+                )}
+                <ManageColumnsToolbar
+                  resourceFields={fields}
+                  defaultColumns={fieldsMetadata}
+                  setColumns={setFields}
+                />
+                {GlobalActionToolbarItems?.length > 0 &&
+                  GlobalActionToolbarItems.map((Action, index) => (
+                    <Action key={index} dataOnScreen={showPagination ? pageData : filteredData} />
```

Comment:
can we use something else as key and not the index?

Reply:
There would have to be some refactoring of GlobalActionToolbarItems or even just the means in which these are passed between components in order to use a reliable key that isn't an index here. I think all of those changes would be out of scope. In general, there is a lot of this practice of iterating over elements versus arrays of objects that render elements. The latter is what we should aim for.